packages/react/src/converters/ReactStackTraceConverter.ts

```diff
@@ -0,0 +1,20 @@
+import { BacktraceStackTraceConverter } from '@backtrace/sdk-core';
+import { JavaScriptEngine } from '@backtrace/sdk-core/src/model/data/JavaScriptEngine';
+import { BacktraceStackFrame } from '@backtrace/sdk-core/lib/model/data/BacktraceStackTrace';
+import { isReact16ComponentStack, parseReact16ComponentStack } from '../helpers/componentStackHelpers';
+
+export class ReactStackTraceConverter implements BacktraceStackTraceConverter {
```

Comment:
how about moving everything from compontentStackHelper here?  I think it does make sense to keep simple and small logic inside one dedicated file.

Reply:
I moved everything over and also updated the tests